### ISSUE TITLE
[ENH] Propagate Folder Permissions to subfolders

### ISSUE THREAD

**Issue by GregStevenson** [oqtane/oqtane.framework#4245]
### Oqtane Info

Version - 5.1.1
Render Mode - N/A 
Interactivity - N/A
Database - All

### Describe the enhancement
When changing permissions on a folder add a checkbox, to propagate the changes to all the subfolders. There probably should also be an optional argument on the server side methods to do this programatically., 

### Anything else?
My understanding is that DNN had this functionality.

**Comment by sbwalker** [comment]
@GregStevenson thank you for submitting this enhancement... it will be considered amongst the other enhancement requests. As a side note, there is no goal to replicate all of the features of DNN in Oqtane - Oqtane is a completely new product with its own vision and product roadmap.

**Comment by mdmontesinos** [comment]
@sbwalker Is there any chance to revisit this issue? Not long ago, a "similar" option was added to page permissions where the updated permissions propagate to all its modules.

<img width="1373" height="703" alt="Image" src="https://github.com/user-attachments/assets/4a50440a-4caa-41cc-9aac-9fa26239e8fb" />

I think it would be nice to have a similar option for folders. For example, in my use case, I allow users to create different folders to better organize their assets, but these folders are created programatically as children from a root assets folder for their team. When they need to update permissions for folders in their team, I need to go one by one updating them, instead of just updating the root assets folder.

**Comment by sbwalker** [comment]
@mdmontesinos so you would like a capability like this for subfolders?

<img width="1521" height="880" alt="Image" src="https://github.com/user-attachments/assets/8c755a39-79e7-4ca5-9043-0e360ac7011a" />

**Comment by mdmontesinos** [comment]
@sbwalker yes, exactly like that would be great. I guess it should also handle subfolders recursively (subfolders with nested folders in them)

**Comment by sbwalker** [comment]
yes, the recursive aspect makes it a bit more challenging than the page/module scenario, but I will investigate
